contracts/axelar-interchain-token-service/src/event.rs

```diff
@@ -0,0 +1,16 @@
+use soroban_sdk::{symbol_short, String, Address, Env};
+
+pub(crate) fn set_trusted_address(env: &Env, chain: String, trusted_address: String) {
+    let topics = (symbol_short!("set"), chain, trusted_address);
```

Comment:
```suggestion
    let topics = (symbol_short!("trusted_address_set"), chain, trusted_address);
```
can you take a look at efficiency difference between short symbols vs normal symbols? The deployment scripts have a `--estimateCost` flag that can give you resource consumption details to compare. It's probably minor, in which case we should just switch to more readable names